Makefile

```diff
@@ -264,6 +266,9 @@ release-snapshot:
 update-kopia-image:
 	@/bin/bash ./build/update_kopia_image.sh $(COMMIT_ID) $(KOPIA_REPO)
 
+update-image-reference:
```

Comment:
When we improve the kopia image update to be a manual workflow, this should change as well. A successful run of the image update workflow should trigger this.